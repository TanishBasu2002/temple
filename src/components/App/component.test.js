// @flow

import React from 'react';
import renderer from 'react-test-renderer';

import Component from './component';

it('renders as expected', () => {
  const component = renderer.create(
    <Component />,
  );
  expect(component.toJSON()).toMatchSnapshot();
});
