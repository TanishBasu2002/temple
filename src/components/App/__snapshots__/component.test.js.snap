// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as expected 1`] = `
<div>
  <h2
    className="MuiTypography-root-1 MuiTypography-title-7"
  >
    Temple
  </h2>
  <button
    className="MuiButtonBase-root-62 MuiButton-root-36 MuiButton-text-38 MuiButton-flat-41"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex="0"
    type="button"
  >
    <span
      className="MuiButton-label-37"
    >
      Copy stuff
    </span>
    <span
      className="MuiTouchRipple-root-65"
    />
  </button>
</div>
`;
